geopandas/tests/test_crs.py

```diff
@@ -330,7 +330,7 @@ def test_dataframe_setitem(self):
         df = GeoDataFrame(
             {"col1": [1, 2], "geometry": arr, "other_geom": arr}, crs=4326
         )
-        df["other_geom"] = arr.to_crs(27700)
```

Comment:
I found `GeoSeries.to_crs` also has this problem. It requires `GeoSeries.crs` is no empty.

Reply:
So, I use `set_crs`.
This would be find.